### PR TITLE
Fix redirection issues in web extension for namespace-specific routes

### PR DESCRIPTION
**Pull request by satyakampandya**
This PR addresses a redirection issue in the Sidekiq Cron web extension where redirects were not consistently pointing to namespace-specific routes. The changes ensure that after performing operations (`enqueue`, `delete`, `enable`, `disable`) on cron jobs, the user is redirected to the appropriate namespace view.

### CODE REVIEW

File: lib/sidekiq/cron/web_extension.rb [hunk 1]
```diff
@@ -56,24 +56,24 @@ def self.registered(app)
 
         # Enqueue cron job.
         app.post '/cron/namespaces/:namespace/jobs/:name/enque' do
-          if job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace])
+          if (job = Sidekiq::Cron::Job.find(route_params[:name], route_params[:namespace]))
```
Comment (markets) [inline lib/sidekiq/cron/web_extension.rb:59]: Is this change necessary? I'd prefer to avoid unnecessary diff 🙏

File: lib/sidekiq/cron/web_extension.rb [hunk 2]
```diff
@@ -56,24 +56,24 @@ def self.registered(app)
...
             job.enque!
           end
-          redirect params['redirect'] || "#{root_path}cron"
+          redirect params['redirect'] || "#{root_path}cron/namespaces/#{route_params[:namespace]}"
```
Comment (markets) [inline lib/sidekiq/cron/web_extension.rb:62]: Maybe we can create a helper to DRY this code? Something like "redirect_with_namespace".

File: lib/sidekiq/cron/web_extension.rb (hunk 1 above)
Comment (satyakampandya) [reply lib/sidekiq/cron/web_extension.rb:59]: Not necessary, just following best practice guidelines, to add clarifying parentheses.

File: lib/sidekiq/cron/web_extension.rb (hunk 2 above)
Comment (satyakampandya) [reply lib/sidekiq/cron/web_extension.rb:62]: Should we implement the `redirect_with_namespace` method inline within the `WebExtension `like this?
```ruby
app.helpers do
  def redirect_with_namespace
    "#{root_path}cron/namespaces/#{route_params[:namespace]}"
  end
end
```
Or would it be better to create a separate helper file?

Comment (markets) [reply lib/sidekiq/cron/web_extension.rb:62]: I think it's fine for now keep it inline. We can create a separate module in the future when more helpers are implemented.

**Review by markets — APPROVED** [review]

**Comment by markets** [comment]
Thanks @satyakampandya 🙌
